src/app/views/provisioning/summary.html

```diff
@@ -51,6 +51,9 @@ <h2 translate>mno_enterprise.templates.dashboard.provisioning.summary.subscripti
           <span ng-show="vm.subscription.product_pricing.pricing_type == 'payg'">
             {{ 'mno_enterprise.templates.dashboard.provisioning.order.pricing_type.payg' | translate }}
           </span>
+          <span ng-show="vm.subscription.product_pricing.pricing_type == 'free'">
+            {{ 'mno_enterprise.templates.dashboard.provisioning.order.pricing_type.free' | translate }}
```

Comment:
Is this locale in the mnoe yaml files?

Reply:
Already done https://github.com/maestrano/mno-enterprise/pull/628/files